### PR TITLE
DO NOT MERGE: Minimal example for representation

### PR DESCRIPTION
**Pull request by ziw-liu**
Produce a source distribution by:
1. Clone the repo
2. Checkout this branch
3. Delete Git by running `rm -rf ./.git`

Then test if the no-git version in the shared drive:
- installation works with a pared-down version
- training and prediction works with the example file

### CODE REVIEW

**Comment by Soorya19Pradeep** [comment]
@ziw-liu , I tried doing a fresh installation. I am getting the following error:

> (dynaclr) soorya.pradeep@SFO-IT0358 viscy % pip install -e ".[visual,metrics]"
> Obtaining file:///Users/soorya.pradeep/Documents/GitHub/Dynaclr/viscy
>   Installing build dependencies ... done
>   Checking if build backend supports build_editable ... done
>   Getting requirements to build editable ... done
>   Preparing editable metadata (pyproject.toml) ... done
> Collecting iohub==0.1.0 (from viscy==0.3.0a1)
>   Using cached iohub-0.1.0-py3-none-any.whl.metadata (6.5 kB)
> INFO: pip is looking at multiple versions of viscy to determine which version is compatible with other requirements. This could take a while.
> ERROR: Could not find a version that satisfies the requirement torch>=2.4.1 (from viscy) (from versions: 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2, 2.2.0, 2.2.1, 2.2.2)
> ERROR: No matching distribution found for torch>=2.4.1

**Comment by ziw-liu** [comment]
@Soorya19Pradeep Seems like you are testing on macOS. Can you share the output of this:

```
python -c "import torch; print(torch.mps.device_count())"
```

Can you also test on Linux?

**Comment by ziw-liu** [comment]
@Soorya19Pradeep also if you are using conda, it's helpful to look at the output of `conda info` to make sure that arm64 is used instead of x86_64.

**Comment by Soorya19Pradeep** [comment]
@ziw-liu , I tried it on an HPC OnDemand session. It does work there.
`conda info` does show x86_64. 

> (dynaclr) soorya.pradeep@SFO-IT0358 viscy % conda info
> 
>      active environment : dynaclr
>     active env location : /opt/miniconda3/envs/dynaclr
>             shell level : 2
>        user config file : /Users/soorya.pradeep/.condarc
>  populated config files :
>           conda version : 4.11.0
>     conda-build version : not installed
>          python version : 3.9.19.final.0
>        virtual packages : __osx=10.16=0
>                           __unix=0=0
>                           __archspec=1=x86_64
>        base environment : /opt/miniconda3  (writable)
>       conda av data dir : /opt/miniconda3/etc/conda
>   conda av metadata url : None
>            channel URLs : https://repo.anaconda.com/pkgs/main/osx-64
>                           https://repo.anaconda.com/pkgs/main/noarch
>                           https://repo.anaconda.com/pkgs/r/osx-64
>                           https://repo.anaconda.com/pkgs/r/noarch
>           package cache : /opt/miniconda3/pkgs
>                           /Users/soorya.pradeep/.conda/pkgs
>        envs directories : /opt/miniconda3/envs
>                           /Users/soorya.pradeep/.conda/envs
>                platform : osx-64
>              user-agent : conda/4.11.0 requests/2.27.1 CPython/3.9.19 Darwin/21.5.0 OSX/10.16
>                 UID:GID : 502:20
>              netrc file : None
>            offline mode : False

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Contrastive learning of cell state dynamics in response to perturbations.
 > The CTC example configs are also tested on macOS with Apple M1 Pro SoCs (macOS 14.7).
 > Apple Silicon users need to make sure that they use
 > the `arm64` build of Python to use MPS acceleration.
+> Tested to work on Linux on the High Performance cluster, and may not work in other environments.
```
Comment (ziw-liu) [inline README.md:13]: This is the same as line 9. Also it won't be clear to the reader what 'the High Performance cluster' is.

Comment (ziw-liu) [reply README.md:13]:
```suggestion
> Other system configurations may not work as intended.
```

**Pull request closed without merging**